clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/input/kafka/KafkaInputUtils.java

```diff
@@ -56,14 +55,15 @@ public static VeniceProperties getConsumerProperties(JobConf config) {
             veniceProperties.clipAndFilterNamespace(KafkaInputRecordReader.KIF_RECORD_READER_KAFKA_CONFIG_PREFIX)
                 .toProperties());
         // Copy the mandatory ssl configs
-        ApacheKafkaUtils.validateAndCopyKafkaSSLConfig(veniceProperties, consumerFactoryProperties);
+        consumerFactoryProperties.putAll(veniceProperties.getAsMap());
```

Comment:
validateAndCopyKafkaSSLConfig was only copying certain configs. With this change, we might be copying in more configs. Will this become an issue?

Reply:
This shouldn't be an issue. In fact, we want to propagate all configurations down to the PubSub adapters so they can pick up what they need. This is necessary because we can't explicitly add configurations for every custom PubSub adapter in the OSS repo.

Reply:
makes sense, thanks.